Marlin/src/lcd/extui/lib/mks_ui/wifi_module.cpp

```diff
@@ -142,172 +151,222 @@ void wifi_reset() {
 }
 
 void mount_file_sys(uint8_t disk_type) {
-  if (disk_type == FILE_SYS_SD) {
+  if (disk_type == FILE_SYS_SD)
     TERN_(SDSUPPORT, card.mount());
-  }
   else if (disk_type == FILE_SYS_USB) {
   }
```

Comment:
You need the curly braces here { }.

`TERN_` evaluated to nothing if SDSUPPORT is false, which would evaluate to this invalid code:

```
if (disk_type == FILE_SYS_SD)
else if (disk_type == FILE_SYS_USB) {
}
```